extended.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  root: true,
+  plugins: ["react", "react-hooks", "jsx-a11y"],
+  extends: [
+    "./index.js",
+    "plugin:react/recommended",
+    "plugin:react-hooks/recommended",
+    "plugin:jsx-a11y/recommended",
+  ],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  rules: {
+    "jsx-a11y/anchor-is-valid": [
+      "warn",
+      {
+        specialLink: ["to"],
+      },
+    ],
+    // don't force .jsx extension
+    "react/jsx-filename-extension": "off",
+    // In TS you must use the Fragment syntax instead of the shorthand
+    "react/jsx-fragments": "off",
+    // This allows props to be spread in JSX
+    "react/jsx-props-no-spreading": "off",
+    // ensure props are alphabetical
+    "react/jsx-sort-props": "error",
+    // We don't need default props on TS components
+    "react/require-default-props": "off",
+    "react/sort-prop-types": "error",
```

Comment:
Do we automatically resolve that error?

Reply:
I think we discussed this some time ago, right?
I didn’t include changes, just moved the existing config to `extended` . So it was there already.
Having said that, I just tested it and my intellij sorts them automatically when the error occurs